.github/pull_request_template.md

```diff
@@ -1,34 +1,30 @@
 Pull request overview
 ---------------------
-Please change this line to a description of the pull request, with useful supporting information including whether it is a new feature, or fixes a defect, a cross reference to any defects addressed in this PR, the type of changes to be made, and whether diffs are expected/justified based on this change.
+ - Fixes #ISSUENUMBERHERE (IF THIS IS A DEFECT)
+ - DESCRIBE PURPOSE OF THIS PULL REQUEST
 
-### Work Checklist
+**NOTE: ENHANCEMENTS MUST FOLLOW A SUBMISSION PROCESS INCLUDING A FEATURE PROPOSAL AND DESIGN DOCUMENT PRIOR TO SUBMITTING CODE**
+
+### Pull Request Author
 Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
  - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
- - [ ] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
-   - Defect: This pull request repairs a github defect issue.  The github issue should be referenced in the PR description
-   - Refactoring: This pull request includes code changes that don't change the functionality of the program, just perform refactoring
-   - NewFeature: This pull request includes code to add a new feature to EnergyPlus
-   - Performance: This pull request includes code changes that are directed at improving the runtime performance of EnergyPlus
-   - DoNoPublish: This pull request includes changes that shouldn't be included in the changelog
+ - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
+ - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
+ - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
+ - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
+ - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
+ - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
+ - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
+ - [ ] If structural output changes, add to output rules file and add OutputChange label
```

Comment:
This is a beautiful thing!

- [ ] If figures are included with documents, include figure name in CMakeLists.txt (e.g., ...\EnergyPlus\doc\input-output-reference\CMakeLists.txt)
- [ ] I'm sure there are other requirements like this ... and other things I don't even know about

Reply:
Thanks @rraustad, I think we can make better use of this checklist as it gets tidied up.  I'll add your figure name entry here as well.

Reply:
And let's not forget those critical PR tags.
- [ ] If table columns are added include OutputChange tag so that PR review process can begin before deadline
- [ ] If IDD is changed include IDDChange tag so that PR review process can begin before deadline

etc...

Reply:
Oh, you already have OutputChange label. Also, should reviewer run develop and this branch using the defect file, or the developer? It's seems the developer would have already done that and could easily provide a plot of the results. We don't want to duplicate effort.

Reply:
So the reason a reviewer would also want to do that is twofold: one it gives someone a chance to evaluate the fix with a fresh set of eyes, and also if the develop branch has moved quite a bit since the PR was issued, the combination of this branch and the new develop could have side effects that cause unit tests to fail or IDFs to fail.  It's a good idea to have a reviewer pull updated develop branch in and build and run.  The reviewer shouldn't have to do any investigating though, just reproducing what was already seen.

Reply:
Ripe for a Wiki page...eventually...nice work.